consultation_analyser/consultations/models.py

```diff
@@ -1,7 +1,43 @@
-# Create your models here.
 from django.db import models
 
 
+class Consultation(models.Model):
+    # May need something better long-term
+    class Organisation(models.TextChoices):
+        DFE = "dept-for-education", "Department for Education"
+        CO = "cabinet-office", "Cabinet Office"
+
+    identifier = models.CharField(max_length=256, blank=False)
+    name = models.CharField(max_length=256, blank=False)
+    organisation_identifier = models.CharField(max_length=256, choices=Organisation.choices, blank=True)
+
+
 class Question(models.Model):
     text = models.CharField(max_length=None)  # no idea what's a sensible value for max_length
     slug = models.CharField(max_length=256)
+    is_closed = models.BooleanField(default=False)
+    is_open = models.BooleanField(default=False)
+    closed_response_options = models.JSONField(default=list)
+    order = models.IntegerField(blank=True, null=True)
+    consultation = models.ForeignKey(Consultation, on_delete=models.CASCADE, null=True)
+
+
+class Respondent(models.Model):
+    # These characteristics might be different for different consultations
+    age_group = models.CharField(max_length=256, blank=True)
+    location = models.CharField(max_length=256, blank=True)
+    # TODO - what is capacity?
+
+
+class Theme(models.Model):
+    label = models.CharField(max_length=256, blank=True)
+    summary = models.TextField(blank=True)
+    keywords = models.JSONField(default=list)
```

Comment:
Do we need any fields beyond "summary" at this point?

Reply:
I think we need keywords at least for the outputs. I think label and summary are the "title" and "description" Kev needs for the prototype. But I won't really know til I've gone through the OVA code. Can remove fields and add when I need them.

Reply:
If these are outputs from the pipeline I guess let's include them